include/veriblock/logger.hpp

```diff
@@ -88,12 +88,12 @@ LogLevel StringToLevel(const std::string&);
 
 //! @private
 #define VBK_LOG(lvl, format, ...)                             \
-  do {                                                        \
+  {                                                           \
     auto& logger = altintegration::GetLogger();               \
     if (logger.level <= lvl) {                                \
       logger.log(lvl, VBK_LOG_FORMAT(format, ##__VA_ARGS__)); \
     }                                                         \
-  } while (0)
+  }
```

Comment:
do not remove these. it is the only way to make VBK_LOG work as expression in C++